htdocs/compta/paiement.php

```diff
@@ -135,7 +135,7 @@
 			{
 				$cursorfacid = substr($key, 21);
 	            $multicurrency_amounts[$cursorfacid] = price2num(trim(GETPOST($key)));
-	            $multicurrency_totalpayment += $multicurrency_amounts[$cursorfacid];
+	            $multicurrency_totalpayment .= $multicurrency_amounts[$cursorfacid];
```

Comment:
This looks stange.  The .= is used to concat string. For a total, we usually want a sum of numeric values, so  +=

Reply:
using += give the issue #14352, tested with .= and issue solved. not sure the real reason behind.

$multicurrency_totalpayment += $multicurrency_amounts[$cursorfacid];

Reply:
The change remove the warning,but it returns a bad value of the total.